internal/usage/usage.go

```diff
@@ -46,6 +46,7 @@ const (
 	DiagnoseOut                     = "Optional output filename, if none given will use diagnose-archive.tar.gz."
 	LogStart                        = "Beginning of the period for which to retrieve logs."
 	LogEnd                          = "End of the period for which to retrieve logs."
+	X509Type                        = "X.509 method by which the provided username is authenticated. Valid values are NONE, MANAGED, or CUSTOMER."
```

Comment:
[nit-style] when we define valid values we tend to add them as a new line, you can see at line 141, we also move this multiline descriptions to the bottom as they mess up with the gofmt comand

Reply:
Oh nice, I missed those ones at the bottom